src/config/commands.xml

```diff
@@ -23,6 +23,7 @@
         <service parent="rokka.command.base"
                  id="rokka.command.user_create"
                  class="RokkaCli\Command\UserCreateCommand"
+                 public="true"
```

Comment:
why do the commands need to be public?

Reply:
I get this, when they're not public:
```
$ bin/console config:dump-reference RokkaClientBundle


 [WARNING] Some commands could not be registered:


In Container.php line 252:

  The "rokka.command.user_create" service or alias has been removed or inlined when the container was compiled. You should either make it public, or stop using the container directly and use dependency injection instead.
```

Reply:
seems we need to enable lazy support or make them public: https://symfony.com/blog/new-in-symfony-3-4-lazy-commands . will do the lazyness thing.